api/src/main/java/edu/wpi/first/shuffleboard/api/util/TypeUtils.java

```diff
@@ -24,4 +25,37 @@ private TypeUtils() {
   public static <T> Function<Object, Optional<T>> optionalCast(Class<T> cls) {
     return value -> optionalCast(value, cls);
   }
+
+  /**
+   * Allows you to filter out members of a subtype from a stream of some base type.
+   * For example, this code:
+   *
+   * <code>
+   *   getComponents().stream()
+   *     .filter(c -> c instanceof Widget)
+   *     .map(c -> (Widget) c)
+   *     .forEach(w -> w.setSource(...))
+   * </code>
```

Comment:
Use: 
```
* <pre>
* {@code
* System.out.println("");
* }
* </pre>
```

Reply:
We typically use HTML `<code>` tags for multi-line code blocks like this

Reply:
Those do not render correctly in JavaDoc.